codegen/src/main/twirl/templates/ScalaClient/Client.scala.txt

```diff
@@ -93,32 +90,6 @@ final class Default@{service.name}Client(settings: GrpcClientSettings)(implicit
 
 }
 
-@mapMethodType(methodType: akka.grpc.gen.MethodType) = {
-  @if(methodType == akka.grpc.gen.Unary) { MethodDescriptor.MethodType.UNARY }
-  @if(methodType == akka.grpc.gen.ClientStreaming) {MethodDescriptor.MethodType.CLIENT_STREAMING }
-  @if(methodType == akka.grpc.gen.ServerStreaming) { MethodDescriptor.MethodType.SERVER_STREAMING }
-  @if(methodType == akka.grpc.gen.BidiStreaming) {MethodDescriptor.MethodType.BIDI_STREAMING }
-}
-
-private object Default@{service.name}Client {
-
-  def apply(settings: GrpcClientSettings)(implicit sys: ClassicActorSystemProvider): @{service.name}Client =
```

Comment:
It looks like removing this object caused MiMa to flag a potential binary incompatibility. I think this might probably be all-right since this object is private anyway, but perhaps double-check this apply method would not be used from user code. If so then we can add a mima exclusion rule to specify this is all-right.

Reply:
i removed it, because it was failing on compilation, 
after i moved method descriptors, there was just apply function which was not used